src/detect-tls.h

```diff
@@ -38,8 +38,7 @@ typedef struct DetectTlsData_ {
     uint16_t ver; /** tls version to match */
     uint32_t flags; /** flags containing match variant (Negation for example) */
     char * subject; /** tls certificate subject substring to match */
-    char * issuerdn; /** tls certificate issuerDN substring to match */
```

Comment:
@jeff wonder if we can do the same for subject/issuerdn as we did for fingerprint?

Reply:
@jlucovsky I think you were meant to be tagged here ^^

Reply:
updated :)